2.7.0/final.md

```diff
@@ -0,0 +1,958 @@
+- Expose `sharedMemPerMultiprocessor` device property to python ([#143119](https://github.com/pytorch/pytorch/pull/143119))
+- Expose remaining sharedMem `cudaDeviceProps` to python ([#143226](https://github.com/pytorch/pytorch/pull/143226))
+- Add range check for embedding_bag on input `index >= 0` of cuda device ([#140791](https://github.com/pytorch/pytorch/pull/140791))
+- Fix linter warnings ([#147386](https://github.com/pytorch/pytorch/pull/147386))
+- Change behavior of pinning memory so it does not init a cuda context if one is not already present ([#145752](https://github.com/pytorch/pytorch/pull/145752))
+- Add cutlass kernel for rowwise scaled mm on SM 10.0 (blackwell) ([#148421](https://github.com/pytorch/pytorch/pull/148421))
+- Add `get_stream_from_external` API for CUDA backend ([#143799](https://github.com/pytorch/pytorch/pull/143799))
+- Update cuDNN-frontend submodule to 1.10.0, used by cuDNN convolution and SDPA integrations ([#145780](https://github.com/pytorch/pytorch/pull/145780))
+
+## MPS
+
+## ROCm
+- Fix TunableOp UTs: Rotating Buffer (#143172)
+- Enable *_load_dwordx4 ISA for BFloat16 and Half. ([#141397](https://github.com/pytorch/pytorch/pull/141397))
+- Fix condition for small tensor tuning ([#144087](https://github.com/pytorch/pytorch/pull/144087))
+
+## XPU
+- Enable FP64 GEMM ([#140677](https://github.com/pytorch/pytorch/pull/140677))
+- Enable Sparse CSR support ([#144722](https://github.com/pytorch/pytorch/pull/144722))
+- Improve XPU Stream implemenation([#141123](https://github.com/pytorch/pytorch/pull/141123),[#141119](https://github.com/pytorch/pytorch/pull/141119),[#142347](https://github.com/pytorch/pytorch/pull/142347))
+- Enable XPU for Inductor MM Triton Kernel Benchmark ([#148237](https://github.com/pytorch/pytorch/pull/148237))
+- Align XPU `convolution_backward` output layout between fake tensor and real output tensor ([#146880](https://github.com/pytorch/pytorch/pull/146880))
+- Improve error handling and reporting in CMake files ([#149353](https://github.com/pytorch/pytorch/pull/149353))
+- Refine `torch.xpu.get_device_properties` API error message ([#144379](https://github.com/pytorch/pytorch/pull/144379))
+- Enable `nested_layer_norm` support for XPU ([#148593](https://github.com/pytorch/pytorch/pull/148593))
+- Generalize `is_big_gpu()` check in Inductor ([#143491](https://github.com/pytorch/pytorch/pull/143491))
+- Allow XPU device in sparse compressed tensor factory functions (#147306)
+
+## Profiler
+- Add optional flag to profiler to toggle external correlations (#143314)
+- Add delimeter in memory vizualizer to show where allocation addr begins (#147461)
+- Add last entry to truncated values in Kineto args (#148576)
+- Add profiler activity for HPU devices (#148182)
+
+
+## AOTDispatcher
+AOTDispatcher is a "middleware" component of torch.compile, responsible for normalizing the graph captured by dynamo
```

Comment:
Why do we describe what it is here?
Same for the other categories below

Reply:
cuz @bdhirsh did extra work 😛  but yes will remove and put under torch.compile